### ISSUE TITLE
Concept: support gridded/raster data in grapher

### ISSUE THREAD

**Issue by larsyencken** [owid/owid-grapher#1578]
## Concept

There are a number of raster map datasets that explain important concepts and give new sources of evidence on existing topics. If we could support raster maps, both in our data catalog and in grapher, then we could more easily discuss these topics with users.

## What would be involved

- We would need to support new data types in the ETL
- We might need a tile engine if there are lot of details (e.g. Mapbox)
- We would need to bring in new raster datasets and work out how to QA them

# Notable gridded datasets

See: [Gridded data](https://www.notion.so/owid/bfee015cdb6b470b9463e06f008b7872?v=ad41b39885f349e8abe498575b000a82) in Notion

## Related ideas

- Basically all gridded datasets could be re-aggregated into national or [sub-national datasets](https://github.com/owid/owid-grapher/issues/1577)

**Comment by danyx23** [comment]
@larsyencken I'm tempted to close this one - it's straightforward enough to create a similar issue if we come up with a good use case for it and it looks like we won't have the appetite in 2023 for this

**Comment by larsyencken** [comment]
This is actually an example of something we won't do in 2023. Closing it is a good idea.
